src/components/aeIdentity/aeIdentity.scss

```diff
@@ -0,0 +1,139 @@
+@import "./../variables";
+
+.ae-identity {
+  top:auto;
+  bottom:10px;
+  box-sizing:border-box;
+  zoom: 1.1;
+  padding:15px 21px;
+  color:white;
+  z-index:5;
+  margin:0 auto;
+  text-align: left;
+  width: 335px;
+  box-shadow: 0 4px 8px 0 rgba(60, 60, 60, 0.1);
+  border-radius: 12px;
+  background-image: linear-gradient(122deg, #6f53a1, $exciting);
+
+  &.active{
+    background-image: linear-gradient(119deg, #f22f70, #42255c);
+  }
+
+  &.collapsed{
+  }
+
+  .footer {
+    margin-top: 18px;
+
+    .buttons {
```

Comment:
this class is never used in the component. we can not assume, that developers will use a class like that in the slot, and if they do, know what it means